Update WSL doc with proper library to install

ryanlai2 opened the pull request:
libomp5-11 cannot be found when doing an apt install. I think should be libomp5

opbld32 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-releasenotes/docs/release-notes.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/release-notes?branch=pr-en-us-45) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C16%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810162004096870-45%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ngVU3BxzzDXFk3aeIcXALC0o0Y%2Beu2xgREexh%2FRG9Ls%3D&st=2018-10-16T20%3A00%3A23Z&se=2018-11-16T20%3A05%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld32 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-releasenotes/docs/release-notes.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/release-notes?branch=pr-en-us-45) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C16%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810162027141648-45%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FDzoTMH6xP1x9VosSvDj7sj5xmeXIOmrpfbeWUXRSXA%3D&st=2018-10-16T20%3A23%3A24Z&se=2018-11-16T20%3A28%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Update gpu-pytorch-wsl.md

libomp5-11 cannot be found when doing an apt install. I think should be libomp5

Commit:
Update gpu-pytorch-wsl.md

ryanlai2 commented:
Will solve this issue : https://github.com/MicrosoftDocs/windows-ai-docs/issues/44

ryanlai2 commented:
This change will get the proper libraries installed so Pytorch DML can work on WSL2
